.github/workflows/quality.yml

```
@@ -0,0 +1,48 @@
+name: Quality Checks
+
+on:
+  push:
+    branches: [ main, docs-release-pipeline ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)